core/aws.py

```diff
@@ -1,29 +1,23 @@
 import json
 import boto3
 from botocore.exceptions import ClientError
+from app import app
+import logging
+
+logging.config.fileConfig('logging.config')
+log = logging.getLogger()
+log.setLevel(app.config['LOG_LEVEL'])
```

Comment:
i dislike this whole block in this file. kind of a chicken vs egg smell here. it's cuz the way we use secrets to set ENV vars. this one i'd just leave the way it was and let it raise that clientError.

i dont like this whole circle dance in this file

Reply:
Reverted